src/styles/card.scss

```diff
@@ -0,0 +1,52 @@
+@import 'variables';
+
+.product-card {
```

Comment:
Use sass nesting for card block.

Reply:
<img width="580" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/302d26a6-83d3-4d43-a704-e2bf36a69e49">